src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+class IsoscelesTrapezoid extends Figure {
+    private final int base1;
+    private final int base2;
```

Comment:
naming with numbers is generally discouraged

Reply:
fixed